src/transformers/models/convnext/configuration_convnext.py

```diff
@@ -100,14 +99,12 @@ def __init__(
         layer_norm_eps=1e-12,
         layer_scale_init_value=1e-6,
         drop_path_rate=0.0,
-        image_size=224,
```

Comment:
In general there shouldn't be a need to change the config file at all for a Flax port